pkg/printers/template_flags.go

```diff
@@ -36,6 +36,10 @@ type GoTemplatePrintFlags struct {
 	TemplateArgument *string
 }
 
+func (f *GoTemplatePrintFlags) AllowedFormats() []string {
+	return []string{"template", "go-template", "go-template-file", "templatefile"}
```

Comment:
We already have a map of supported formats, see `ToPrinter` method. We should have a single source of truth with these.

Reply:
Done.

Reply:
Also modified in:
jsonpath_flags.go
customcolumn_flags.go